CHANGES.md

```diff
@@ -1,6 +1,11 @@
 Change Log
 ==========
 
+### 1.11 - 2015-07-01
+
+* Breaking changes
+  * PerformanceDisplay (private API) no longer takes style options.  It is now styled with app-overridable CSS.  Use `scene.debugShowFramesPerSecond` to enable it as before.
```

Comment:
I would change this to something like "The performance statistics displayed by setting `scene.debugShowFramesPerSecond` to `true` can now be styled using the `cesium-performanceDisplay` CSS classes in `shared.css`.